src/components/workspace/index.tsx

```diff
@@ -204,13 +204,15 @@ class Workspace extends React.Component<WorkspaceProps, {}> {
       ) {
         return (
           <div className="editor-content">
+            <div className="editor-title">Prepend Code</div>
```

Comment:
Plus what does "prepend" even mean? Can we go with a more non-technical term that a student can associate with => such as "available functions"

Reply:
I removed the title for the editors

Reply:
Removing the title will make it hard to ascertain why the prepend code editor is even there in the first place. 

Possible to move the prepend code portion out to one of the tabs/hide it on a button press? 
Or if thats too difficult, why not call the prepend code, "Available functions" ?

Reply:
The prepend is resizable so it can be hidden by resizing it.

The prepend can contain any code, such as constants and functions, so I don't think calling it "available functions" is ok, maybe "Helpers"? I think we can use a `<Collapse>` to tag the editor as well as introduce a mechanism to hide the prepend editor.

Reply:
Okay that sounds good